packages/api/src/logger/index.ts

```diff
@@ -0,0 +1,200 @@
+import pino, {
+  BaseLogger,
+  DestinationStream,
+  LevelWithSilent,
+  LoggerOptions,
+  redactOptions,
+} from 'pino'
+
+/**
+ * Determines if log environment is development or test
+ *
+ * Boolean
+ */
+const isDevelopment =
+  process.env.NODE_ENV === 'development' || process.env.NODE_ENV === 'test'
+
+/**
+ * Determines if log environment is production by checking if not development
+ *
+ * Boolean
+ */
+const isProduction = !isDevelopment
+
+/**
+ * Determines if logs should be prettified.
+ *
+ * Typically if logging to a transport stream or in production
+ * logs will not be prettified.
+ *
+ * In fact, the pino developers note:
+ * "We recommend against using pino-pretty in production,
+ * and highly recommend installing pino-pretty as a development dependency."
+ * See: https://github.com/pinojs/pino-pretty#programmatic-integration
+ *
+ * One exception to this rule may be Netlify functions logging.
+ * Its function logging output readability can benfeit from pretty-printing.
+ *
+ * Boolean
```

Comment:
I'm not a TS expect, but isn't one of the benefits of typescript that you can "enforce" the type?

So you'd get something like:

`export const isPretty : Boolean = isDevelopment`

Or is that lazily assigned by the first assignment?

Reply:
> Or is that lazily assigned by the first assignment?

> I'm not a TS expert

Neither am I ... this construction seemed to work, but other reviews here would be helpful to know if this this a proper way of says `isPretty must be a boolean but it defaults to try if you are in dev`

Reply:
If you want to use jsdoc to specify the type here, this is what you do
```
/**
 * @type {boolean}
 */
```

But yeah, TS will infer the type to be `boolean`, and nothing else. No need to explicitly state it (i.e. this is not needed `export const isPretty: boolean = isDevelopment`)